File: nandazman/2 Even Fibonacci Numbers/solution.py
def fiboEvenSum(n):
  total = 0

  num1 = 1
  num2 = 1

  while num2 < n:
    num2 = num1 + num2
    num1 = num2 - num1

    if num2 % 2 == 0:
      total = total + num2
  
  return total

print(fiboEvenSum(4000000))

File: nandazman/2 Even Fibonacci Numbers/solution.js
function fiboEvenSum(n) {
  // You can do it!
  let total = 0;
  let sum = 0;

  let num1 = 1;
  let num2 = 1;
  while(sum < n) {
    sum = num1 + num2;
    num1 = num2;
    num2 = sum;
    if(sum % 2 === 0) {
      total = total + sum;
    }
  }
  return total;
}

fiboEvenSum(10);
